src/Slider/Slider.js

```diff
@@ -0,0 +1,198 @@
+// @flow
+
+import * as React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { View } from 'react-native';
+import MultiSlider from 'react-native-multi-slider';
+
+import Marker from './Marker';
+import SliderParts from './SliderParts';
+import { Text } from '../Text';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+const TRACK_HEIGHT = 4;
+
+type Props = {|
+  +type: 'multi' | 'single',
+  +minValue: number,
+  +maxValue: number,
+  +startValue: number,
+  +endValue?: number,
+  +label: string | React.Node,
+  +numOfParts?: number,
+  +snapped?: boolean,
+  +step?: number,
+  +customMarker?: React.Node,
+  +sliderLength?: number,
+  +onValuesChange?: () => void,
```

Comment:
Can `onValuesChange` also return actual slider value?

For example on implementation below (and in current mobile app) the **Time of stay** value should change during dragging.

<img width="356" alt="screenshot 2019-01-10 at 15 18 59" src="https://user-images.githubusercontent.com/2660330/50974322-45a7b680-14eb-11e9-9a17-a7a64332cc92.png">